src/GraphQL.php

```diff
@@ -177,53 +177,6 @@ public static function promiseToExecute(
         }
     }
 
-    /**
-     * @deprecated Use executeQuery()->toArray() instead
-     *
-     * @param string|DocumentNode $source
-     * @param mixed               $rootValue
-     * @param mixed               $contextValue
-     * @param mixed[]|null        $variableValues
-     *
-     * @return Promise|mixed[]
-     *
-     * @codeCoverageIgnore
-     */
-    public static function execute(
-        SchemaType $schema,
-        $source,
-        $rootValue = null,
-        $contextValue = null,
-        $variableValues = null,
-        ?string $operationName = null
-    ) {
-        trigger_error(
-            __METHOD__ . ' is deprecated, use GraphQL::executeQuery()->toArray() as a quick replacement',
-            E_USER_DEPRECATED
-        );
-
-        $promiseAdapter = Executor::getPromiseAdapter();
-        $result         = self::promiseToExecute(
-            $promiseAdapter,
-            $schema,
-            $source,
-            $rootValue,
-            $contextValue,
-            $variableValues,
-            $operationName
-        );
-
-        if ($promiseAdapter instanceof SyncPromiseAdapter) {
-            $result = $promiseAdapter->wait($result)->toArray();
-        } else {
-            $result = $result->then(static function (ExecutionResult $r): array {
-                return $r->toArray();
-            });
-        }
-
-        return $result;
-    }
-
     /**
      * @deprecated renamed to executeQuery()
```

Comment:
Even worse 😆

Reply:
That should be in https://github.com/webonyx/graphql-php/pull/823, hm? 🙃